### PR TITLE
Add some changes in construct_from_raw function.

### PR DESCRIPTION
**Pull request by MariaHovakimyan2000**

### CODE REVIEW

File: pycognaize/document/field/numeric_field.py [hunk 1]
```diff
@@ -29,11 +29,13 @@ def __init__(self,
...
                  group_key: str = None,
                  confidence: Optional[float] = -1.0,
-                 group_name: str = None
+                 group_name: str = None,
+                 scale: int = None
                  ):
         super().__init__(name=name, tags=tags, value=value,
                          group_key=group_key, confidence=confidence,
                          group_name=group_name)
+        self.scale = scale
```
Comment (hovhannes-zohrabyan) [inline pycognaize/document/field/numeric_field.py:39]: this line is not needed

File: pycognaize/tests/test_common/test_lazy_dict.py [hunk 2]
```diff
@@ -55,9 +55,10 @@ def test___iter__(self):
             ['60f554497883ab0013d9d906', '60b76b3d6f3f980019105dac',
              '60f53e967883ab0013d9c6f9', '60f5260c7883ab0013d9c184',
              '60215310dbf28200120e6afa', '62eb8e6b28d7ca0012ec8288',
-             '62eb8e6b28d7ca0012ec8288_error', '632c61fc86d52800197d03f3', '6405fdd7420ed0001184e4f3'])
+             '62eb8e6b28d7ca0012ec8288_error', '632c61fc86d52800197d03f3', '6405fdd7420ed0001184e4f3',
```
Comment (hovhannes-zohrabyan) [inline pycognaize/tests/test_common/test_lazy_dict.py]: Fix indentation

File: pycognaize/tests/test_document/test_field/test_numeric_field.py [hunk 3]
```diff
@@ -3,29 +3,42 @@
...
+        self.numeric_field_scale_2 = self.data_with_scale['output_fields']['v_other_receivables_bs__current'][0]['scale']
+        self.numeric_field_value_3 = self.data_with_scale['output_fields']['v_other_receivables_bs__current'][0]['tags'][0]['ocrValue']
 
         self.pages_1 = {1: create_dummy_page(page_n=1)}
         self.pages_2 = {1: create_dummy_page(page_n=1),
                         2: create_dummy_page(page_n=2)}
 
+        self.snapshot = Snapshot(RESOURCE_FOLDER + '/snapshots')
+        self.document = self.snapshot.documents['63d4486c0e3fe60011fe3a75']
+        # self.document = Document.from_dict(self.data_with_scale, data_path='/home/maria/Desktop/pycognaize/pycognaize/tests/resources/snapshots')
```
Comment (hovhannes-zohrabyan) [inline pycognaize/tests/test_document/test_field/test_numeric_field.py]: Remove commented lines

**Review by hovhannes-zohrabyan — COMMENTED** [review]
Please update according to the comments

File: pycognaize/document/field/numeric_field.py (hunk 1 above)
Comment (hovhannes-zohrabyan) [reply pycognaize/document/field/numeric_field.py:39]: Ignore this @MariaHovakimyan2000

**Review by hovhannes-zohrabyan — APPROVED** [review]

**Review by davidcognaize — APPROVED** [review]

**Pull request merged**